docs/components/sensor/fake.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Create a Custom Sensor Model"
```

Comment:
wrong title I think. This is not creating a custom one but rather one that always sends the same readings